backend/app/db/supabase/auth.py

```
@@ -1,25 +1,24 @@
+
+async def login_with_oauth(provider: str):
     try:
         result = supabase_client.auth.sign_in_with_oauth({
             "provider": provider,
-            "options": {
-                "redirect_to": os.getenv("SUPABASE_REDIRECT_URL", "http://localhost:3000/home")
-            }
         })
         return {"url": result.url}
     except Exception as e:
         raise Exception(f"OAuth login failed for {provider}: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)